src/LibraryManager/Manifest.cs

```diff
@@ -20,6 +20,7 @@ namespace Microsoft.Web.LibraryManager
     /// </summary>
     public class Manifest
     {
+        private static readonly string[] _supportedVersions = { "1.0" };
```

Comment:
I think this is good for now, though maybe we should think of using a single Version object (rather than a string) and using the value as "highest supported version". I assume we will always support lower than current versions? Something we should probably discuss, but again - I'm totally fine with the string array for now.

Reply:
Yes, we should discuss the compatibility model going forward. I just implemented the simplest support for now.